package/kedro_viz/models/graph.py

```diff
@@ -497,6 +515,20 @@ def __post_init__(self, data_node: DataNode):
             with dataset._fs.open(load_path, **dataset._fs_open_args_load) as fs_file:
                 self.plot = json.load(fs_file)
 
+        if data_node.is_metric_node():
+            if not dataset._exists():
+                return
+            

```

Comment:
Too many blank lines - I guess we might have to update the linters for viz, because we check this stuff on kedro core 😉